exec/bigmachine.go

```diff
@@ -632,6 +637,9 @@ func (w *worker) Run(ctx context.Context, req taskRunRequest, reply *taskRunRepl
 			if task != nil {
 				task.Error(errors.Recover(err))
 			}
+			// Allow log/printf output to flush to journalctl, without this,
+			// it appears that some log output is lost.
+			time.Sleep(time.Second)
```

Comment:
Should we instead add a `log.Sync` method that does a best effort syncing (e.g. checks the underlying `Writer` for a `Sync` or maybe does this sleep thing).  It's still not terribly reliable, but maybe useful to call in known-fatal scenarios(?).  (Presumably, we're failing to sync to stdout as opposed to `journalctl` not picking it up).